File: backend/product-services/src/main/java/vn/edu/iuh/fit/dhktpm17a/productservices/repositories/ProductRepository.java
package vn.edu.iuh.fit.dhktpm17a.productservices.repositories;

import org.springframework.data.mongodb.repository.MongoRepository;
import vn.edu.iuh.fit.dhktpm17a.productservices.entities.Product;

public interface ProductRepository extends MongoRepository<Product, String> {
}

File: backend/product-services/src/main/java/vn/edu/iuh/fit/dhktpm17a/productservices/repositories/ProductRepository.kt
package vn.edu.iuh.fit.dhktpm17a.productservices.repositories

import org.springframework.data.jpa.repository.JpaRepository
import vn.edu.iuh.fit.dhktpm17a.productservices.entities.Product

interface ProductRepository : JpaRepository<Product, String> {
}